src/test/signalk/SKNMEA2000ParserTest.cpp

```diff
@@ -83,5 +83,35 @@ TEST_CASE("SKNMEA2000Parser: Basic tests") {
     CHECK( update.getEnvironmentDepthBelowKeel() == 40.8 );
     CHECK( update.getEnvironmentDepthTransducerToKeel() == 1.2 );
   }
-}
 
+  SECTION("127250: VESSEL True HEADING RAPID ") {
+    SetN2kTrueHeading(msg, 0, 180);
```

Comment:
The value 180 is in radians here. So it's not what you think it is ;) Use `SKDegToRad(180)`